code/game/objects/items/flamethrower.dm

```diff
@@ -71,7 +71,7 @@
 		. += "+lit"
 
 /obj/item/flamethrower/afterattack(atom/target, mob/user, flag)
-	. = ..()
+	. = ..() | AFTERATTACK_PROCESSED_ITEM
```

Comment:
oh man this is nasty

Reply:
its a flag

Reply:
yeah but people hardly know what . = ..() is at the best of times

Reply:
ill change it to `. |=` i suppose